### DISCUSSION TITLE
SuiteAdvanced Global Search Menu

### DISCUSSION THREAD

**Discussion by Travis048** [burkybang/SuiteAdvanced-Feedback#99] [Q&A] [answered]
I've noticed that since the last NetSuite update the dropdown for the global search is not working again. You can click on it and it will show the selections of record types but your selection doesn't actually work. This definitely isn't super important but it is a handy feature if you get a chance. Thanks!

**Comment by burkybang** [comment, marked as answer]
Thanks for reporting this. I've created issue #100 from this discussion.

**Comment by Travis048** [comment]
Thank you!

On Tue, Sep 12, 2023 at 9:21 AM Adam Smith ***@***.***> wrote:

> Thanks for reporting this. I've created issue #100
> <https://github.com/burkybang/SuiteAdvanced-Feedback/issues/100> from
> this discussion.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/burkybang/SuiteAdvanced-Feedback/discussions/99#discussioncomment-6978986>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7PYVNSUQUVOHLEYMTZK3K3X2BOVBANCNFSM6AAAAAA4UZ73AY>
> .
> You are receiving this because you authored the thread.Message ID:
> <burkybang/SuiteAdvanced-Feedback/repo-discussions/99/comments/6978986@
> github.com>
>


-- 
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
--
*Travis Corsi*
*Operations Manager*
*Jendco Safety Supply, Inc**.*

Office: 614-449-9600

Cell: *740-404-7748*

www.jendcosafety.com - ***@***.***
